pandas-stubs/io/gbq.pyi

```diff
@@ -12,10 +12,10 @@ def read_gbq(
     dialect: str | None = ...,
     location: str | None = ...,
     configuration: dict[str, Any] | None = ...,
-    credentials=...,
+    # Google type, not available
+    credentials: Any = ...,
     use_bqstorage_api: bool | None = ...,
```

Comment:
according to docs, `use_bqstorage_api` can't be `None`

Reply:
The default value is `None`, so it clearly is accepted.

Reply:
Probably worthwhile to create an issue in the `pandas` repo